### PR TITLE
LITE-29335 Create schedule task for daily upload

### PR DESCRIPTION
**Pull request by jonatrios**

### CODE REVIEW

File: connect_bi_reporter/webapp.py [hunk 1]
```diff
@@ -24,3 +36,54 @@ def on_startup(cls, logger, config):
         # here we are going to add migration based on alembic.
         create_db(config)
         logger.info('Database created...')
+        client = get_extension_owner_client(logger)
+        try:
+            installation = get_extension_owner_installation(client)
```
Comment (maxipavlovic) [inline connect_bi_reporter/webapp.py:41]: Didn't fully understand this part as this seems to be similar `connect_bi_reporter/events.py`

Comment (maxipavlovic) [reply connect_bi_reporter/webapp.py:41]: 1. Okay, didn't know that. It sounds a bit weird as the Extension owner has it's own installation as well.

Anyway, code seems similar and could be reused?

Comment (jonatrios) [reply connect_bi_reporter/webapp.py:41]: agree with last comment, will update it!

Comment (jonatrios) [reply connect_bi_reporter/webapp.py:36]: updated!

**Pull request merged**